GHzDACs/jump_table.py

```diff
@@ -0,0 +1,480 @@
+# Author: Daniel Sank
+# Created: July 2013
+
+import numpy as np
+from util import littleEndian
+
+IDLE_NUM_BITS = 15
+IDLE_MIN_CYCLES = 0
+IDLE_MAX_CYCLES = (2 ** IDLE_NUM_BITS) - 1
```

Comment:
Wouldn't hurt to briefly explain what these are by referring to bytes in the jump table write documentation. That may be more distracting than helpful though... not sure.